### DISCUSSION TITLE
Default behavior of linkerd - Timeout and Retries

### DISCUSSION THREAD

**Discussion by vishaaldubey** [linkerd/linkerd2#4394] [General] [answered]
What is the default behavior of linkerd in terms of timeout and retries if we don't setup service profile.

**Comment by grampelberg** [comment, marked as answer]
There are no timeouts or retries if you don't have service profiles setup.

**Reply by vishaaldubey** [reply]
Thank you! If we setup service profile for few services and not for others, will it still not use any timeouts and retries for those for which service profile is not setup? I saw in documentation if unspecified, the default timeout is 10 seconds.

**Reply by grampelberg** [reply]
The default timeout is 10s, but only when you apply a service profile (as it is the default on a profile). Profiles only match the service you've associated them with, so no other traffic will receive the default.

**Comment by wmorgan** [comment]
See also https://github.com/linkerd/website/issues/721
